src/pages/groupComparison/GroupComparisonMutationsTabPlot.tsx

```diff
@@ -136,7 +136,7 @@ export default class GroupComparisonMutationsTabPlot extends React.Component<
                 );
                 return (
                     <>
-                        <h3>
+                        <h3 data-test="LollipopPlotTitle">
```

Comment:
there's probably an easy way to get this h3 without using data-test and if so, that's a better way to go.  we only want to use data-test when it's totally necessary.   here for example, some parent div already has a data-test attribute and you can then look for h3 inside of that.